helm/ffc-demo-claim-service-infra/values.yaml

```diff
@@ -1,23 +1,45 @@
 namespace: ffc-demo
 
-subscriptionId: 
 serviceBusResourceGroupName: sndadpinfrg1401
 serviceBusNamespaceName: sndadpinfsb1401
 
 asoAnnotations:
   serviceoperator.azure.com/reconcile-policy: manage
 
 namespaceQueues:
-  - name: payment
+  - name: claim
+    owner: 'no'
+    roleAssignments:
+      - roleName: 'QueueReceiver'
   - name: calculation
+    roleAssignments:
+      - roleName: 'QueueSender'
 
 namespaceTopics:
   - name: schedule   
+    roleAssignments:
+      - roleName: 'TopicSender'
 
 postgresResourceGroupName: sndadpdbsrg1401
```

Comment:
should be var or abstracted/defaulted?

Reply:
These need to be explicitly provided by developer. In the backend on ASO helm library, we will assign proper role assignments based on the role provided here.

Reply:
can you discuss with ganesh how to abstract? they need to be vars too for example as it's env specific

Reply:
@DaRoszko done